app/src/NativeBase/Components/ChoicesList.tsx

```diff
@@ -8,6 +8,10 @@ import MaterialIcons from 'react-native-vector-icons/MaterialIcons'
 
 export interface ChoicesListChoice {
   text: string
+  color: string
```

Comment:
I think these options would be better set for the `ChoicesList` overall rather than for each `ChoicesListChoice` -- because we'd expect all items in one list to share the same colours?  (at least that's what the current designs look like)

Reply:
That would also avoid the need for the `.map` on line 27 of ProfileContainer.tsx, where you're having to repeat the same styling options again and again

Reply:
Actually, just thinking about this some more, this would be more reusable if we didn't have to pass through a whole set of colour options at all, but could just pass through one or two options (e.g. a `colour` e.g. 'purple' and/or a `variant`/`type` property e.g. 'bold') -- and then the colours and font weight etc is set in `ChoicesList` based on that?

That would make it much easier in the future if we want to change e.g. the purple that's used in every 'purple' version of a list.  Or the font sizing, font weight, etc that's used in a 'bold' version of this.  We'd only have to update `ChoicesList`.  Otherwise (the way this is working at the moment) we'd have to find every different place where the ChoicesList is used for a purple/bold list and update all these individual settings possibly many times over.

Reply:
Ah, excellent idea, much nicer! 💯  Will make those changes, thanks @joefhall! :)